.github/workflows/backtrace-commit-fix.yml

```
@@ -0,0 +1,22 @@
+name: 'Backtrace issue commit correction'
+on:
+  issues:
+    types:
+      - opened
+jobs:
+  backtrace_commit_correction:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      ISSUE: ${{ github.event.issue.html_url }}
+    if: |
+      contains(github.event.issue.labels.*.name, 'backtrace.io')
+    steps:
+      - name: Modify issue contents
+        env:
+          BODY: ${{ github.event.issue.body }}
+        id: fix
+        run: |
+          changed_body=$(echo "$BODY" | sed 's#<code>\([^<]*\)</code>#\1#g')
```

`<code>\([^<]*\)</code>` may match more than just the hash. I think it's saver to narrow it to `[0-9a-f]{7}` (or possibly `{7,8}` for future longer hashes).

```suggestion
          changed_body=$(echo "$BODY" | sed 's#<code>\([0-9a-f]{7}\)</code>#\1#g')
```

---

I see https://github.com/OpenRCT2/OpenRCT2/issues/19501 was changes while checkout out an issue. Did you update this manually?

---

Yea